packages/dx-react-grid-bootstrap4/src/templates/paging-panel/page-size-selector.test.jsx

```diff
@@ -1,24 +1,24 @@
 import * as React from 'react';
-import { mount } from 'enzyme';
+import { shallow } from 'enzyme';
 import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
 import { PageSizeSelector } from './page-size-selector';
 
 describe('PageSizeSelector', () => {
   describe('#render', () => {
-    const mountPageSizeSelector = ({
+    const shallowPageSizeSelector = ({
```

Comment:
Why do we use a function to create a component with default configuration and not extracting defaults to a variable (e.g. `defaultProps`) like in other tests?